internal/server/v2/facet/golden/series_facet/series_facet.json

```diff
@@ -33,58 +33,66 @@
               ]
             },
             {
-              "facet_id": "217147238",
+              "facet_id": "1151455814",
               "observations": [
                 {
-                  "value": 5
+                  "value": 28
```

Comment:
@shifucun what is the value here?

Reply:
This is the number of observations for this facet. @chejennifer with the change to add earliest and latest date, I wonder how do we put "earliest_data", "latest_date" and "obs_count" here together that still conforms the V2 observation response.

One option is to have 

`
[{"date": "2010"}, {"date": "2020"} 
`
But count is hard to fill in.

Any ideas?

Reply:
sorry, I missed this, but we've chatted offline about updating the response structure